src/components/events/Goals/style.scss

```diff
@@ -0,0 +1,17 @@
+#CampaignGoals, #CampaignGoalsExample {
+  overflow-x: scroll;
+}
+
+#CampaignGoals:-webkit-scrollbar, #CampaignGoalsExample:-webkit-scrollbar {
+  display: none;
+}
+
+.CampaignGoals__container {
+  @extend %clearfix;
+  width: 1050px;
```

Comment:
Has this been tested on different screen sizes? How do we see it working responsively?